ons_alpha/jinja2/templates/pages/bulletins/bulletin_page.html

```diff
@@ -1,17 +1,10 @@
-{% extends "templates/base_page.html" %}
+{% extends "base_page.html" %}
```

Comment:
@islas104 can you please rebase your branch off latest `main`?

the bulletin page template was updated in #14

Reply:
It is good to see a rebase, but now this change throws away Nicks work. Can you please tweak this so you only add your changes, rather than undo the template work